build_oci_go_plugin.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+#Do grunt work
+
+#if [[ ! -d ./node_modules ]]; then
+#  echo "dependencies not installed try running: npm install"
+#  exit 1
+#fi
+
+#./node_modules/.bin/grunt
```

Comment:
@japatel Could you explain why did you comment grunt task?

Reply:
On my Mac, Wanted to build the linux code only using docker golang:1.13. 
NOTE: The original build.sh is intact.

Here are the steps to build I used on my Mac.

1. ./build.sh <-- Build the grant and the Mac binary.
2. Build the linux binary only. Hence removed the grunt from build_oci_go_plugin.sh  script.
docker run --rm -v ~/oci-grafana-plugin:~/oci-grafana-plugin -v ~/go:~/go -w ~/go/src/oci-grafana-plugin -e GOOS=linux -e GOARCH=amd64 -e GOPATH=~/go/ golang:1.13 ~/go/src/oci-grafana-plugin/build_oci_go_plugin.sh

This way do not have to do too much to add node.js and go components to docker golang:1.13 and was able to build the linux driver only.

Reply:
I see thanks, I was it's build.sh